docs/Contribute.md

```diff
@@ -0,0 +1,32 @@
+# Contribution Guidelines
+
+Thanks for taking the time to contribute!
+
+The following is a set of guidelines for contributing to SecDevLabs. These are just guidelines, not rules, so use your best judgement and feel free to propose changes to this document in a pull request! 😃
+
+## So... I built this app, it looks great, but how can I send it to SecDevLabs?
+
+Well, a good place to start is to make sure you can deploy your app with Docker-Compose. All the apps in SecDevLabs are built with it in order to ensure you can get get right into the action! ✓
```

Comment:
It would be great if we had a good reference here on how to build a docker-compose file and environment.

Reply:
When I started learning Docker-Compose, [this] article helped me alot! What do you think, @rafaveira3 ?

[this]: https://www.tutorialspoint.com/docker/docker_compose.htm

Reply:
Hmm... @nettoclaudio , Would you mind helping us with a good reference for this one? 😃